src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -73,6 +73,9 @@
         <service id="form.type.datetime" class="Symfony\Component\Form\Extension\Core\Type\DateTimeType">
             <tag name="form.type" />
         </service>
+        <service id="form.type.dateinterval" class="Symfony\Component\Form\Extension\Core\Type\DateIntervalType">
```

Comment:
A service for this type is not needed as it has no dependency, we just use FQCN since 2.8. Btw almost all the others have been deprecated in 3.1.